Resources/config/routing_website.yml

```diff
@@ -7,3 +7,8 @@ sulu_headless.api.search:
     path: /api/search
     defaults:
         _controller: sulu_headless.controller.search:getAction
+
+sulu_headless.api.snippet_area:
+    path: /api/snippet/{area}
```

Comment:
not sure if we should use `/api/snippet-areas/{area}` here. maybe we want to provide an api for loading a snippet by its id in the future? 🤔

Reply:
Yes, sounds good 👍

Reply:
@nnatter i would say that would be another API `/api/snippets/{id}` or  `/api/snippets/{key}` would keep the path `/api/snippet-areas/{area}`

Reply:
@wachterjohannes currently it's `/api/snippet/{area}` and @nnatter suggested using `/api/snippet-areas/{area}` 🙂

Reply:
then @nnatter and i want the same 😅